src/main/java/org/kuali/kfs/module/purap/document/PaymentRequestDocument.java

```diff
@@ -1239,6 +1241,14 @@ public boolean answerSplitNodeQuestion(final String nodeName) throws Unsupported
         if (nodeName.equals(PurapWorkflowConstants.IS_DOCUMENT_AUTO_APPROVED)) {
             return isAutoApprovedIndicator();
         }
+        if (nodeName.equals(TREASURY_MANAGER_ROUTE_NODE)) {
+            /*
+             * CU Customization KFSPTS-34074
+             * This allows canceled and disapproved PREQ documents that were done before 2/9/2025,
```

Comment:
This may need rewording to say what specifically the change allows, or to alternatively replace "allows" with "fixes".

Reply:
ok